tools/qontract_cli.py

```diff
@@ -478,6 +478,25 @@ def bot_login(ctx, cluster_name):
     print(f"oc login --server {server} --token {token}")
 
 
+@get.command()
+@click.argument('account_name')
+@click.pass_context
+def aws_creds(ctx, account_name):
+    settings = queries.get_app_interface_settings()
+    secret_reader = SecretReader(settings=settings)
+    accounts = queries.get_aws_accounts()
+    accounts = [a for a in accounts if a['name'] == account_name]
```

Comment:
For a future iteration, I think we should do the filtering on the server side. It'll be faster and lead to less code around here.

Reply:
i agree

Reply:
done